hex/loader.go

```diff
@@ -0,0 +1,30 @@
+package hex
+
+import (
+	lua "github.com/yuin/gopher-lua"
+)
```

Comment:
This can be inlined: `import lua "github.com/yuin/gopher-lua"`.

Reply:
DONE